### PR TITLE
Update automate-tests.mdx

### PR DESCRIPTION
**Pull request by aravindraveendran7**
Added `npm ci` as step 1 to avoid any kind of warning while installing dependencies.

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3809"></a>

### CODE REVIEW

File: pages/docs/contributing/automated-tests.mdx [hunk 1]
```diff
@@ -4,7 +4,7 @@ import DocsLayout from "../../../components/layouts/DocsLayout.js";
 
 We use [Playwright](http://playwright.dev) for writing automated end to end (e2e) tests.
 
-1. Install Playwright dependencies `npx playwright install --with-deps`
```
Comment (eddiejaoude) [inline pages/docs/contributing/automated-tests.mdx:7]: This step is still required, we discussed adding the `npm ci` step before this existing step not replacing it

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
Please add new pre step not replace existing steps

File: pages/docs/contributing/automated-tests.mdx (hunk 1 above)
Comment (aravindraveendran7) [reply pages/docs/contributing/automated-tests.mdx:7]: okay, my bad. Just to understand better, ' npm ci` should be step 1 followed by other steps correct? and do we add the warning as well along with step 1?

Comment (eddiejaoude) [reply pages/docs/contributing/automated-tests.mdx:7]: Yes please...

1. install dependencies with `npm ci`
2.Install Playwright dependencies `npx playwright install --with-deps`
3. Run tests `npm run test`

Comment (aravindraveendran7) [reply pages/docs/contributing/automated-tests.mdx:7]: Thank you @eddiejaoude

**Comment by aravindraveendran7** [comment]
closing this open PR, as new PR #4529 has been raised

**Pull request closed without merging**